data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -124,22 +124,37 @@ def _setup_etl_instance_and_run_extract(
         used to retrieve data in order to force that method to retrieve the fixture
         data. A basic version of that patching is included here for classes that can use it.
         """
-        with mock.patch("data_pipeline.utils.requests") as requests_mock:
-            zip_file_fixture_src = (
-                self._DATA_DIRECTORY_FOR_TEST / self._SAMPLE_DATA_ZIP_FILE_NAME
-            )
+
+        with mock.patch(
+            "data_pipeline.utils.requests"
+        ) as requests_mock, mock.patch(
+            "data_pipeline.etl.score.etl_utils.get_state_fips_codes"
+        ) as mock_get_state_fips_codes:
             tmp_path = mock_paths[1]
+            if self._SAMPLE_DATA_ZIP_FILE_NAME is not None:
+                zip_file_fixture_src = (
+                    self._DATA_DIRECTORY_FOR_TEST
+                    / self._SAMPLE_DATA_ZIP_FILE_NAME
+                )
 
-            # Create mock response.
-            with open(zip_file_fixture_src, mode="rb") as file:
-                file_contents = file.read()
+                # Create mock response.
+                with open(zip_file_fixture_src, mode="rb") as file:
+                    file_contents = file.read()
+            else:
+                with open(
+                    self._DATA_DIRECTORY_FOR_TEST / self._SAMPLE_DATA_FILE_NAME,
+                    "rb",
+                ) as file:
+                    file_contents = file.read()
             response_mock = requests.Response()
             response_mock.status_code = 200
             # pylint: disable=protected-access
             response_mock._content = file_contents
             # Return text fixture:
             requests_mock.get = mock.MagicMock(return_value=response_mock)
-
+            mock_get_state_fips_codes.return_value = [
```

Comment:
Hmm. Will this cause problems with the "expected states in data" logic? Or at least be redundant with it? 

We may want the mock to just return all states/DC/territories, but then have the code for `_get_instance_of_etl_class` continue to use a more limited list using all this code: 

```
# Find out what unique state codes are present in the test fixture data.
        states_expected_from_fixtures = {
            x[0:2] for x in self._FIXTURES_SHARED_TRACT_IDS
        }

        # Set values to match test fixtures
        etl_class.EXPECTED_MISSING_STATES = [
            x
            for x in TILES_CONTINENTAL_US_FIPS_CODE
            + TILES_ALASKA_AND_HAWAII_FIPS_CODE
            if x not in states_expected_from_fixtures
        ]
        etl_class.PUERTO_RICO_EXPECTED_IN_DATA = False
        etl_class.ISLAND_AREAS_EXPECTED_IN_DATA = False
        etl_class.ALASKA_AND_HAWAII_EXPECTED_IN_DATA = True
```

Alternatively, we can proceed with the approach in this line and mocking the return value of  `get_state_fips_codes` to just the codes we expect from the fixtures, but then we should delete all that code that's now redundant in `_get_instance_of_etl_class`.